modules/hadoop/src/main/tut/README.md

```diff
@@ -0,0 +1,48 @@
+# Hadoop module for PureConfig
+
+Adds support for selected [Hadoop](http://hadoop.apache.org/) classes to PureConfig.
+
+## Add pureconfig-hadoop to your project
+
+In addition to [core PureConfig](https://github.com/pureconfig/pureconfig), you'll need:
+
+```scala
+libraryDependencies += "com.github.pureconfig" %% "pureconfig-hadoop" % "0.9.0"
+```
+
+Also, `pureconfig-hadoop` depends on `hadoop-common` with `provided` scope. It's mean that you should explicitly add a dependency on `hadoop-common` or any other Hadoop library who depends on `hadoop-common`. Usually it would be something like this:
+
+```scala
+libraryDependencies += "org.apache.hadoop" % "hadoop-client" % "3.0.0"
```

Comment:
Do you know if the `Path` class is binary compatible across Hadoop v2 and v3? 

I'm curious since I'm stuck on v2. If they're not compatible, it might nice to mention that here.

Reply:
I checked - `Path` is binary compatible. But overall there is no strong guarantee (SHOULD and SHELL instead of MUST) for binary 2 and 3 major releases. It may be a problem for some cases, but at the current moment we could ignore this.

Reply:
That seems entirely reasonable. Thanks for checking into it.